content/_guides/cloaks.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Cloaks 
+category: using
+---
+
+A cloak is a custom hostmask that replaces your client hostname or IP address,
+as shown in IRC operational messages or `/whois`.
+
+For example, the hostmask `jsmith@10-10-32-1.res.dsl.example.com` might appear
+with a cloak as `jsmith@user/jsmith`.
+
+A cloak can help other users and bots identify you simply and consistently. It
+can also help limit the exposure of your IP address on the network, although
+it does not necessarily provide anonymity or prevent the discovery of your IP
+address.
+
+As an individual, you can get a generic user cloak in the form of
+`user/username`. If you are involved with a well-known project or community,
+you may be able to get a cloak that indicates your affiliation with that
+group.
+
+You may also sometimes see some temporary or `gateway` cloaks associated with
+gateway services, such as when using our
+[tor hidden service](/guides/connect#accessing-liberachat-via-tor). These only
+last for the duration that the user is connected through that service and help
+users of the service be differentiated from each other. You cannot request
+these cloaks.
+
+## Generic user cloaks
+
+You can request a generic user cloak from the bot in `#libera-cloak`. The
+cloak will be in the form of `user/<NickServ username>`. Registration with
+`NickServ` is required.
+
+### Requesting a self-service user cloak
+
+Please note: This process may change in the future.
+
+1. [Register with NickServ](/guides/registration) if you have not already
+   done so. You will also need to be [identified](cloaks/).
```

Comment:
Should this be linking to https://libera.chat/guides/registration#logging-in ?

Reply:
lol yes oops